### ISSUE TITLE
Skype crash just after start

### ISSUE THREAD

**Issue by qlahcim** [flathub/com.skype.Client#230]
On Linux Mint 21.3 the latest version of the flathub Skype always crash just after start!!!
Complete reinstallation solve this problem only temporary...soon, or later same problem with immediate crash after start.

Any workaround or solution???

**Comment by michalkvasnicka** [comment]
Any response??? Current status of Skype is completely unusable on LinuxMInt and other Ubuntu 22.04 based distros??!!

**Comment by Hyperchaotic** [comment]
This is a long shot but try to check with Flatseal what sockets are enabled. It should have X11 enabled and Wayland disabled.

**Comment by rudolfkastl** [comment]
I have crash issues on Fedora 40 with all flatpak skype versions newer than 8.113.0.210.  8.113.0.210 is the last version that works. Note: if you downgrade make sure you do not have parts of the crashed newer skype version running before you try out the old version to verify.

**Comment by nerijus** [comment]
Does not crash here On F40. Version 8.119.0.201. Make sure to delete all `.skype*` and `.Skype*` dirs in your home dir.

**Comment by nerijus** [comment]
Yes, it works. I had a problem before that settings were not saved and I had to remove `~/.Skype*` (not `~/.var/app/...`!) dir, and then settings were saved and Skype starts more than once.

**Comment by michalkvasnicka** [comment]
Latest version 8.125.0.201, considered as fully working on Ubuntu, crash on Linux Mint 21.3 after start, again, and again ...??!!

**Comment by rudolfkastl** [comment]
I got the issue tracked down to the permissions of:
~/.var/app/com.skype.Client/data/keyrings/default.keyring

They are set to 0400 with the owner and group of my user. If i set it to 0644 skype stops crashing on second startup after saving the credentials.

**Comment by nerijus** [comment]
On my system `~/.var/app/com.skype.Client/data/keyrings/default.keyring` permissions are 0600 and it works OK.

**Comment by rudolfkastl** [comment]
> On my system `~/.var/app/com.skype.Client/data/keyrings/default.keyring` permissions are 0600 and it works OK.

Thank you for your reply, actually i could not reproduce making it "not crash" by changing keyring permissions... unfortunately it keeps on crashing and i might have just hit a "lucky condidition" where it worked after changing those permissions. It keeps crashing 10 out of 10 tries to start it unless i remove ~/.var/app/com.skype.Client/* . If i remove the whole config the skype client starts again without crashing 10/10 times.

**Comment by rudolfkastl** [comment]
@qlahcim yes, after i remove:  "~/.var/app/com.skype.Client/*" everything works again.
